deploy/iso/minikube-iso/arch/x86_64/package/cni-plugins/cni-plugins.hash

```diff
@@ -3,3 +3,5 @@ sha256 8589670f7f9b211a351dfcd211d4fe0b961d77283a7415443dc188f3dbf05668  v0.6.0.
 sha256 92c7599918be0a720ac020f137cdeac746dfa03da6b26e08a37132c5728c091f  v0.7.5.tar.gz
 sha256 9d1526ed965ac6562fd95a931ab2346b3c5efd58c2f569038ba3c530f7e66472  v0.8.5.tar.gz
 sha256 bd682ffcf701e8f83283cdff7281aad0c83b02a56084d6e601216210732833f9  cni-plugins-linux-amd64-v0.8.5.tgz
+sha256 c86c44877c47f69cd23611e22029ab26b613f620195b76b3ec20f589367a7962  v1.1.1.tar.gz
```

Comment:
v1.1.1 should go in cni.hash

Reply:
That hashes file has two hashes for each version the linux-amd64 hash and the source code hash, so I added both as well. But I did forget to update the CNI hash which I have resolved. Do you know if both hashes are needed in the first file?

Reply:
it shouldn't be needed, i switch it from building from source, to just grabbing the binaries, which is why the format changed. you should just need the `cni-plugins-linux` formatted one

Reply:
Thanks for the clarification, removed the source code hash